mkdocs_rss_plugin/util.py

```diff
@@ -139,6 +139,28 @@ def build_url(self, base_url: str, path: str, args_dict: dict = None) -> str:
             url_parts[4] = urlencode(args_dict)
         return urlunparse(url_parts)
 
+    def get_value_from_dot_key(self, data: dict, dot_key: Any) -> Any:
```

Comment:
Why `Any` and not `str`?

Reply:
`dot_key: Any` because during the unittest, I found you send bool as dot_key, it was `str` before the failure from unittest.
ref: https://github.com/Guts/mkdocs-rss-plugin/actions/runs/7482036200/job/20372837842#step:7:242
![image](https://github.com/Guts/mkdocs-rss-plugin/assets/13083856/d436b5d9-a5da-4c02-9b07-311f4ed570e5)

`-> Any` as dict value might be any type.

Reply:
maybe should alter the test_build_no_git.py itself ?

Reply:
Hmm I can't remember the use cas right now and I don't have access to my dev setup. I suppose that's here to allow end-user disable it.

Could you change for `Union[bool, str]` please? If possible, I prefer to avoid `Any` which is, IHMO, a convenient workaround when a variable's type is too fuzzy. I remember that I used it intensively at TypeScript beginning and years later we were still celaning it up :sweat_smile:

Reply:
done